src/test/regress/sql/subselect.sql

```diff
@@ -517,3 +517,44 @@ fetch backward all in c1;
 
 commit;
 --end_ignore
+
+-- Ensure that ORCA correctly builts Query dxl tree and all attributes inside
+-- the nested SubLink, which are referenced in GROUP BY clause, are processed
+-- during query normalization. The inner part of SubPLan should contain only t.j
+-- start_ignore
+DROP TABLE IF EXISTS t;
```

Comment:
The queries should be like the nearest code - in lowercase.

Reply:
fixed